.clinerules

```diff
@@ -40,6 +41,19 @@ Follow these rules when you write code:
 - Implement accessibility features on elements. For example, a tag should have a tabindex="0", aria-label, on:click, and on:keydown, and similar attributes.
 - Use consts instead of functions, for example, "const toggle = () =>". Also, define a type if possible.
 - Do not code within the `page.tsx` file in Next.js App Router. Instead, create a separate `XXXPage` component and write all code there.
+- Refer to existing implementation patterns and file structures (e.g., how other pages import modules) to determine the most optimal approach.
+- Follow the `tsconfig.json` paths settings and always use the correct alias for import paths.
+- Always align data fetching responsibilities with the component's role:
+  - If data should be fetched on the server, delegate it to a server component.
+  - Use client-side fetching only when necessary, ensuring performance and UX considerations.
+
+#### Component Implementation Guidelines
+
+- Avoid using `default export`; always use `named export`.
+- Group all files that are exclusive to a component in a single folder and re-export them from an `index.ts` file.
```

Comment:
Since we'll turn off biome rule, This line should be removed 👍🏻 

```suggestion
```

- https://github.com/liam-hq/liam/pull/862